### ISSUE TITLE
Articles that are not published can be viewed

### ISSUE THREAD

**Issue by ajrbyers** [revistajs/revista#23]
Following on from #21, articles that:

- Do not have a publication date or
- Are not in the published stage

Should not be available to read even if the Issue is published.

**Comment by MartinPaulEve** [comment]
Actually, this may be "by design"...

Articles that are not published but that have been accepted should be viewable for early-DOI assignment purposes. But they should not reveal the actual content of the article.

**Comment by ajrbyers** [comment]
Yeah I remember that, but Im thinking more about the issue listing and the banner rotator 👍
